package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "20.5.4",
+  "version": "20.5.5",
```

Comment:
You don't need to add this for patch changes, once you merge, it'll automatically do it in our setup

Reply:
Reverted back to the previous version.